core/src/components/chip/test/a11y/chip.e2e.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ directions: ['ltr'], modes: ['ios', 'md'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
```suggestion
configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Our test infra defaults to testing both md and ios